addons/l10n_ch/models/account_invoice.py

```diff
@@ -19,7 +19,7 @@ class AccountMove(models.Model):
     l10n_ch_isr_subscription_formatted = fields.Char(compute='_compute_l10n_ch_isr_subscription', help="ISR subscription number your company or your bank, formated with '-' and without the padding zeros, to generate ISR report.")
 
     l10n_ch_isr_number = fields.Char(compute='_compute_l10n_ch_isr_number', store=True, help='The reference number associated with this invoice')
-    l10n_ch_isr_number_spaced = fields.Char(compute='_compute_l10n_ch_isr_number', help="ISR number split in blocks of 5 characters (right-justified), to generate ISR report.")
+    l10n_ch_isr_number_spaced = fields.Char(compute='_compute_l10n_ch_isr_number', store=True, help="ISR number split in blocks of 5 characters (right-justified), to generate ISR report.")
```

Comment:
???  Does not that change the model?

Reply:
Is this to make the reconcile possible even with ISR having spaces ?

Reply:
here I only change the field non-store to store because "l10n_ch_isr_number" and "l10n_ch_isr_number_spaced" both have the same compute method so both fields need the same 
store parameter either store or non-store.

Reply:
@fclementic2c No, everything about that has been merged before.  

@jva-odoo You need to have two functions then somehow (maybe sharing another method), because we can not change the store option in stable.

Reply:
@jco-odoo ok 👍

Reply:
@jva-odoo Let me know when it is done.